lambify/bin/_build

```diff
@@ -1,18 +1,12 @@
 #!/bin/sh
 set -e
 
-echo '== Simulate SAM Build Directory =='
-RESOURCE="RailsLambda"
-SAM_TEMP=$(mktemp -d -t sam.XXXXXX)
-rm -rf ./.lamby ./.aws-sam
-cp -r . "$SAM_TEMP"
-mkdir -p ./.lamby
-cp -r "$SAM_TEMP" "./.lamby/$RESOURCE"
-pushd "./.lamby/$RESOURCE"
-
-echo '== Cleaning Dev Dependencies =='
-rm -rf ./.bundle \
-       ./vendor/bundle
+if [ "$CI" != "true" ]; then
+  echo "== Cleaning dev dependencies for local deploy. Run ./bin/setup again afterward! =="
+  rm -rf ./.bundle \
+         ./vendor/bundle \
+         ./node_modules
+fi
```

Comment:
Removed all that stuff to create the `.lamby` directory. Added a simple check when you are not `CI/CD` doing the deploy to clean up your dev dependencies. See further below on what was needed in `./bin/_setup` for the inverse of this work.